README.md

```diff
@@ -11,3 +11,7 @@ A repository of deep learning examples.
 ## Examples
 
 - [YoloV5](yolov5/)
+        - `cd yolov5`
```

Comment:
These instructions don't work.
- no main.py (i guess you mean prepare_dataset)
- explicity state pip install command
- `no module dataset` error

Reply:
I don't want the README.md to be exact instructions